actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -11,7 +11,7 @@ module TagHelper
     # that Trix will write to on changes, so the content will be sent on form submissions.
     #
     # ==== Options
-    # * <tt>:class</tt> - Defaults to "trix-content" which ensures default styling is applied.
+    # * <tt>:class</tt> - Defaults to "trix-content" which ensures default styling is applied. Passing an argument overrides "trix-content", removing the default styles.
```

Comment:
@jacobherrington I went ahead and updated the API per your recommendation.

Reply:
What about more pointed wording?

```suggestion
    # * <tt>:class</tt> - Defaults to "trix-content" so that default styles will be applied. Setting 
        this to a different value will prevent default styles from being applied.
```

There is also another occurrence on [line 58](https://github.com/rails/rails/blob/adc5eb66f861a98da32f7c682155360d5b9a4a52/actiontext/app/helpers/action_text/tag_helper.rb#L58) that should be changed to match.